flame/star/star_model_tester.py

```
@@ -54,13 +58,28 @@ def __init__(self,
-                if not use_dp:
-                    flame = StarModel(**kwargs).flame
-                else:
-                    flame = StarLocalDPModel(**kwargs).flame
-                results_queue.append(flame.final_results_storage)
+                try:
+                    if not use_dp:
+                        flame = StarModel(**kwargs).flame
+                    else:
+                        flame = StarLocalDPModel(**kwargs).flame
+                    results_queue.append(flame.final_results_storage)
+                except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)